src/security_constraints/github_security_advisory.py

```diff
@@ -15,13 +15,12 @@
 
 LOGGER = logging.getLogger(__name__)
 
-
 QUERY_TEMPLATE = string.Template(
     "{"
     "securityVulnerabilities("
     " first: $first"
     " ecosystem:PIP"
-    " severities:$severities"
+    " severities:[$severities]"
```

Comment:
Given that "severities" is a list of str, how does it work when it goes inside brackets here?

Also, are severities case sensitive? (I did not check.)